pkg/utils/filesystem.go

```diff
@@ -71,9 +72,9 @@ func WriteToComposeFile(tempFilePath string, debug bool) bool {
 	errors.CheckErr(err, 203, "")
 
 	if debug == true {
-		fmt.Printf("==> %s structure is: \n%s\n\n", tempFilePath, string(marshalledData))
+		logr.Infof("==> %s structure is: \n%s\n\n", tempFilePath, string(marshalledData))
 	} else {
-		fmt.Println("==> environment structure written to " + tempFilePath)
+		logr.Infoln("==> environment structure written to " + tempFilePath)
```

Comment:
There's a couple of places throughout where the logging uses the formatter vs string concatenation.  Here's an example where both are used.  Nothing wrong with it,  but might be worth standardising.  I personally prefer the formatted version.

Reply:
Sure thing, i totally agree.